py3status/modules/speedtest.py

```diff
@@ -108,8 +108,9 @@
 """
 
 from json import loads
-import time
 import threading
+import time
```

Comment:
Use `from time import time`.

Reply:
```
tests = ["download", "upload"]
for x in tests:
    if x not in self.placeholders:
        self.command += " --no-{}".format(x)
```
I can't have no-upload and no-download together, this is invalid params combo.